pom.xml

```diff
@@ -97,7 +95,7 @@
             <dependency>
                 <groupId>org.slf4j</groupId>
                 <artifactId>slf4j-simple</artifactId>
-                <version>${slf4j.version}</version>
+                <version>${dep.slf4j.version}</version>
```

Comment:
You don't need to redeclare as it's already defined in dependencyManagement in airbase.